docs/basics/daemon.md

```diff
@@ -95,6 +96,12 @@ docker restart cross-seed # Restart the daemon
 docker logs cross-seed # view the logs
 ```
 
+### Unraid
+
+If you are running `cross-seed` on Unraid, it runs in [daemon mode by default](../tutorials/unraid.md#automationscheduling) and no changes are needed.
```

Comment:
Feel free to give me your thoughts on this change
```suggestion
If you are running `cross-seed` on Unraid you can install the "Community App" from ambipro's repository, this is our official container.

By default, the container runs in [daemon mode by default](../tutorials/unraid.md#automationscheduling) and no changes are needed.

We recommend you head over to the [Unraid Tutorial](../tutorials/unraid.md) for specifics and documentation for the container.

Reply:
There's a little text repetition in the second sentence. I cleaned it up in 509594a